sd-journalist/sd-process-display

```diff
@@ -111,7 +128,7 @@ if __name__ == '__main__':
     d.show()
 
     if sigfile != "":
-        # tyvm https://stackoverflow.com/questions/6215690/how-to-execute-a-method-automatically-after-entering-qt-event-loop
+        # tyvm https://stackoverflow.com/questions/6215690/how-to-execute-a-method-automatically-after-entering-qt-event-loop  # noqa: E501
```

Comment:
hmm? noqa?

Reply:
Yes, noqa on E501 permits the extra-long line. I'd _much_ rather skip linting on a URL than break the URL up over multiple lines in order to satisfy the linter!